third_party/move/tools/move-formatter/tests/aptos_framework_case/coin.fmt.move

```diff
@@ -0,0 +1,1443 @@
+    #[expected_failure(abort_code = 0x5000A, location = Self)]
+    public entry fun deposit_frozen(account: signer)
+        acquires CoinInfo, CoinStore {
+        let account_addr = signer::address_of(&account);
+        account::create_account_for_test(account_addr);
+        let (burn_cap, freeze_cap, mint_cap) = initialize_and_register_fake_money(
+            &account, 18, true
+        );
+
+        let coins_minted = mint<FakeMoney>(100, &mint_cap);
+        freeze_coin_store(account_addr, &freeze_cap);
+        deposit(account_addr, coins_minted);
+
+        move_to(
+            &account,
+            FakeMoneyCapabilities {burn_cap, freeze_cap, mint_cap,}
+        );
+    }
+
+    #[test(account = @0x1)]
+    public entry fun deposit_widthdraw_unfrozen(account: signer)
+        acquires CoinInfo, CoinStore {
+        let account_addr = signer::address_of(&account);
+        account::create_account_for_test(account_addr);
+        let (burn_cap, freeze_cap, mint_cap) = initialize_and_register_fake_money(
+            &account, 18, true
+        );
+
+        let coins_minted = mint<FakeMoney>(100, &mint_cap);
+        freeze_coin_store(account_addr, &freeze_cap);
+        unfreeze_coin_store(account_addr, &freeze_cap);
+        deposit(account_addr, coins_minted);
+
+        freeze_coin_store(account_addr, &freeze_cap);
+        unfreeze_coin_store(account_addr, &freeze_cap);
+        let coin = withdraw<FakeMoney>(&account, 10);
+        burn(coin, &burn_cap);
+
+        move_to(
+            &account,
+            FakeMoneyCapabilities {burn_cap, freeze_cap, mint_cap,}
+        );
+    }
+
+    #[test_only]
+    fun initialize_with_aggregator(account: &signer) {
+        let (burn_cap, freeze_cap, mint_cap) = initialize_with_parallelizable_supply<FakeMoney>(
+            account,
+            string::utf8(b"Fake money"),
+            string::utf8(b"FMD"),
+            1,
+            true
+        );
+        move_to(
+            account,
+            FakeMoneyCapabilities {burn_cap, freeze_cap, mint_cap,}
+        );
+    }
+
+    #[test_only]
+    fun initialize_with_integer(account: &signer) {
+        let (burn_cap, freeze_cap, mint_cap) = initialize<FakeMoney>(
+            account,
+            string::utf8(b"Fake money"),
+            string::utf8(b"FMD"),
+            1,
+            true
+        );
+        move_to(
+            account,
+            FakeMoneyCapabilities {burn_cap, freeze_cap, mint_cap,}
+        );
+    }
+
+    #[test(framework = @aptos_framework, other = @0x123)]
+    #[expected_failure(abort_code = 0x50003, location = aptos_framework::system_addresses)]
+    fun test_supply_initialize_fails(framework: signer, other: signer) {
+        aggregator_factory::initialize_aggregator_factory_for_test(&framework);
+        initialize_with_aggregator(&other);
+    }
+
+    #[test(framework = @aptos_framework)]
+    fun test_supply_initialize(framework: signer)
+        acquires CoinInfo {
```

Comment:
Same issue, at the first glance, I thought `acquires CoinInfo` is a statement in the function.